_unittest_ironpython/run_unittests.py

```diff
@@ -26,7 +26,7 @@
 
 args_env = os.environ.get("RUN_UNITTESTS_ARGS", "")
 parser = argparse.ArgumentParser()
-parser.add_argument("--test-filter", "-t", default="test_*.py", help="test filter")
+parser.add_argument("--test-filter", "-t", default="test_41*.py", help="test filter")
```

Comment:
```suggestion
parser.add_argument("--test-filter", "-t", default="test_*.py", help="test filter")
```
You forgot to reset the filter.